common/rpc/interceptor/telemetry.go

```diff
@@ -186,7 +189,9 @@ func (ti *TelemetryInterceptor) UnaryIntercept(
 	handler grpc.UnaryHandler,
 ) (interface{}, error) {
 	methodName := api.MethodName(info.FullMethod)
-	metricsHandler, logTags := ti.unaryMetricsHandlerLogTags(req, info.FullMethod, methodName)
+	nsName := MustGetNamespaceName(ti.namespaceRegistry, req)
```

Comment:
as we discussed, let's not do ns read through here. If namespace is not available in the request, we can use `namespace.EmptyName`